src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -58,19 +60,29 @@ interface Props extends RelayProps {
 }
 
 export class Message extends React.Component<Props, any> {
-  renderAttachmentPreviews(attachments) {
+  renderAttachmentPreviews(attachments: Props["message"]["attachments"]) {
+    const previewAttachment = attachmentID => {
+      const attachment = attachments.find(({ id }) => id === attachmentID)
+      SwitchBoard.presentMediaPreviewController(
+        this,
+        attachment.download_url,
+        attachment.id,
+        attachment.file_name.split(".").reverse()[0]
+      )
+    }
+
     return attachments.map(attachment => {
       if (attachment.content_type.startsWith("image")) {
         return (
-          <ImagePreviewContainer>
-            <ImagePreview imageAttachment={attachment} />
+          <ImagePreviewContainer key={attachment.id}>
```

Comment:
@mzikherman This was another of the warnings you were getting. When adding a list of child components you need to specify the `key` prop with a unique value for each child, preferably a stable value (not an index). See https://facebook.github.io/react/docs/lists-and-keys.html